tests/lxd/test_lxd_backend.cpp

```diff
@@ -857,9 +857,9 @@ TEST_F(LXDBackend, posts_expected_data_when_creating_instance)
                              "\"limits.cpu\":\"2\","
                              "\"limits.memory\":\"3145728\","
                              "\"security.secureboot\":\"false\","
-                             "\"user.meta-data\":\"#cloud-config\\nLuke: Jedi\\n\\n\","
-                             "\"user.user-data\":\"#cloud-config\\nVader: Sith\\n\\n\","
-                             "\"user.vendor-data\":\"#cloud-config\\nSolo: Scoundrel\\n\\n\""
+                             "\"user.meta-data\":\"#cloud-config\\nLuke: Jedi\\n\","
+                             "\"user.user-data\":\"#cloud-config\\nVader: Sith\\n\","
```

Comment:
Not related to this PR, corresponding changes of `emit_cloud_config` code change.